card/db/migrate_core_cards/20141119001955_make_symlinks_relative.rb

```diff
@@ -2,7 +2,7 @@
 
 class MakeSymlinksRelative < Card::Migration
   def up
-    Cardio.paths['files'].each do |file_path| 
+    Card.paths['files'].each do |file_path| 
```

Comment:
Most places we could do this as you suggested.  In some cases, Card isn't loaded yet and we'll still use Cardio, but I tried to make sure all the ones that could be changed were.